tests/cli/RunCest.php

```diff
@@ -128,7 +128,7 @@ public function runPhpUnitXmlReportsInStrictMode(CliGuy $I)
         $I->seeInThisFile('<?xml');
         $I->seeInThisFile(
             '<testsuite name="dummy" tests="6" assertions="3" errors="0" failures="0" skipped="0"'
-            . ' useless="0" time='
+            . ' time='
         );
         $I->seeThisFileMatches('/<testsuite name="AnotherCest" file=".*?AnotherCest.php"/');
         $I->seeThisFileMatches('/<testsuite name="AnotherTest" file=".*?AnotherTest.php"/');
```

Comment:
Please remove another `useless="0"` on line 137 below.

Reply:
Good catch. The dockerized test suite isn't working on my end, so I can't easily check if other tests are failing, I'll wait for the CI.